pom.xml

```diff
@@ -128,7 +128,7 @@
     <!-- ant-assembly-maven-plugin is only used as a plugin -->
     <ant.assembly.version>2.0.6</ant.assembly.version>
     <nuxeo.ftest.version>1.8.3-SNAPSHOT</nuxeo.ftest.version>
-    <nuxeo.java.client.version>3.0.0-SNAPSHOT</nuxeo.java.client.version>
+    <nuxeo.java.client.version>3.0.0-RC1</nuxeo.java.client.version>
 
```

Comment:
please also remove exclude for snapshot dep detection later in the file

<exclude>org.nuxeo.client:nuxeo-java-client</exclude>

Reply:
I will remove this commit, I would rather keeping enforcer mechanism in order to be sure to release the client and not having a SNAPSHOT or a RC in Nuxeo.